pkg/apis/metal3/v1alpha1/baremetalhost_types.go

```diff
@@ -124,6 +124,9 @@ type BMCDetails struct {
 	// The name of the secret containing the BMC credentials (requires
 	// keys "username" and "password").
 	CredentialsName string `json:"credentialsName"`
+
+	// A boolean to skip certificate validation when true
```

Comment:
Maybe we could say:

`DisableCertificateVerification disables verification of server certificates when using HTTPS to connect to the BMC. This is required when the server certificate is self-signed, but is insecure because it allows a man-in-the-middle to intercept the connection.`